### PR TITLE
chore: use errors.New to replace fmt.Errorf with no parameters

### PR DESCRIPTION
**Pull request by ChengenH**
1. remove repetitive words
2. use errors.New to replace fmt.Errorf with no parameters

### CODE REVIEW

File: auction-dutch/chaincode-go/smart-contract/auction.go [hunk 1]
```diff
@@ -8,7 +8,8 @@ import (
 	"bytes"
 	"crypto/sha256"
 	"encoding/json"
-	"fmt"
+    "errors"
+    "fmt"
```
Comment (bestbeforetoday) [inline auction-dutch/chaincode-go/smart-contract/auction.go:12]: This indentation looks to be spaces rather than tabs, as used by gofmt.

**Review by bestbeforetoday — COMMENTED** [review]
Other than a minor indentation issue, looks good to me.

File: auction-dutch/chaincode-go/smart-contract/auction.go (hunk 1 above)
Comment (ChengenH) [reply auction-dutch/chaincode-go/smart-contract/auction.go]: Thanks, I have fixed it

**Review by bestbeforetoday — APPROVED** [review]
Thank you for the contribution

**Pull request merged**